task-sdk/src/airflow/sdk/definitions/baseoperator.py

```diff
@@ -899,6 +899,11 @@ def say_hello_world(**context):
     # Defines if the operator supports lineage without manual definitions
     supports_lineage: bool = False
 
+    # If True, the Rendered Template fields will be overwritten in DB after execution
+    # This is useful for Taskflow decorators that modify the template fields during execution like
+    # @task.bash decorator.
+    overwrite_rtif_after_execution: bool = False
```

Comment:
Good, this will allow it to be extended instead of making changes to individual ones